### PR TITLE
Update events list to include Solid World November and December

### PR DESCRIPTION
**Pull request by ckuesel**

### CODE REVIEW

**Comment by Vinnl** [comment]
@justinwb Do you have the Netlify credentials? Or maybe @jaxoncreed?

> 8:23:52 PM: ---------------------------------------------------------------------
>   UNSUPPORTED BUILD IMAGE
> 
>   The build image for this site uses Ubuntu 16.04 Xenial Xerus, which is no longer supported.
> 
>   To enable builds for this site, select a later build image at the following link:
>   https://app.netlify.com/sites/solidproject-dot-org/settings/deploys#build-image-selection
> 
>   For more details, visit the build migration guide:
>   https://answers.netlify.com/t/please-read-end-of-support-for-xenial-build-image-everything-you-need-to-know/68239
>   ---------------------------------------------------------------------

image

File: pages/solid-events.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
...
+|2022-12-08| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | Solid World December will be held at 10:00 a.m. ET/1600 CET and focus on a number of Solid Community updates, including the formation of a Solid W3C working group. Register for the event on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127). |
+|2022-11-10| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | [Solid World November recording](https://vimeo.com/770472401)<br> The BBC Research & Development team presented their new watch party product, [BBC Together + Data Pod](https://www.bbc.co.uk/taster/pilots/together-pod).|
```
Comment (jaxoncreed) [inline pages/solid-events.md:20]:
```suggestion
|2022-11-10| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | [Solid World November recording](https://vimeo.com/770472401)<br>The BBC Research & Development team presented their new watch party product, [BBC Together + Data Pod](https://www.bbc.co.uk/taster/pilots/together-pod).|
```

**Review by jaxoncreed — COMMENTED** [review]
I'm submitting this review with a minor change to retrigger netlify. For some reason, manually re-running it after upgrading isn't working.

**Comment by jaxoncreed** [comment]
@Vinnl Sorry about that. Netlify has been upgraded, so we're all good now.

File: pages/solid-events.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |------|-------|-----------|-----------------------------|
-|2022-11-10| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | Solid World November will be held at 10:00 a.m. ET/1600 CET and focus on the BBC's new watch party product, [BBC Together + Data Pod](https://www.bbc.co.uk/taster/pilots/together-pod).  Register for the event on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127). |
+|2022-12-08| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | Solid World December will be held at 10:00 a.m. ET/1600 CET and focus on a number of Solid Community updates, including the formation of a Solid W3C working group. Register for the event on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127). |
```
Comment (Vinnl) [inline pages/solid-events.md:19]: Small change for consistent time formats :)

```suggestion
|2022-12-08| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | Solid World December will be held at 10:00 a.m. ET/16:00 CET and focus on a number of Solid Community updates, including the formation of a Solid W3C working group. Register for the event on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127). |
```

**Review by Vinnl — APPROVED** [review]
Thanks @jaxoncreed!

**Pull request merged**